src/main/java/org/sagebionetworks/web/client/widget/login/LoginWidgetViewImpl.java

```diff
@@ -54,14 +54,14 @@ public void postLogin() {
 	}
 
 	private static native void _createSRCLogin(Element el, LoginWidgetViewImpl loginWidgetView,
-	   String googleSSORedirectUrl, SynapseContextProviderPropsJSNIObject wrapperProps) /*-{
+	   String ssoRedirectUrl, SynapseContextProviderPropsJSNIObject wrapperProps) /*-{
 		try {
 			function sessionCallback() {
 				loginWidgetView.@org.sagebionetworks.web.client.widget.login.LoginWidgetViewImpl::postLogin()();
 			}
 
 			var props = {
-				googleRedirectUrl : googleSSORedirectUrl,
+				ssoRedirectUrl : ssoRedirectUrl,
```

Comment:
In the latest version of SRC, the LoginPage component has a prop named ssoRedirectUrl, and it will append the appropriate OAuth provider string based on what provider the user selects to log in with.